src/pages/vendor/Category.tsx

```
@@ -171,28 +299,64 @@ export default function Category() {
+                  const selectedItem = e.target.value;
+                  if (selectedItem && !formData.foodlist.includes(selectedItem)) {
+                    setFormData({ ...formData, foodlist: [...formData.foodlist, selectedItem] });
+                  }
+                }}
                 className="w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-1 focus:ring-brand-primary focus:border-brand-primary"
-                placeholder="Describe this category"
-              />
+              >
+                <option value="">Select a food item</option>
+                {categoryMenu?.map((item: any) => {
+                  return <option value={item?._id}>{item?.name}</option>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)